### PR TITLE
{compiler}[GCCcore/8.3.0] LLVM v10.0.0

### PR DESCRIPTION
**Pull request by Falkor**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Falkor** [comment]
Test report by @ULHPC
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
iris-003 - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 3.7.4
See https://gist.github.com/a2d09b8f5aece7ca9c49938afdad3ce4 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node375.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/aaf25121bc7d06ddac3ad11600c0c908 for a full test report.

File: easybuild/easyconfigs/l/LLVM/LLVM-10.0.0-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+dependencies = [
+    ('ncurses', '6.1'),
+    ('zlib', '1.2.11'),
+]
+
+configopts = '-DBUILD_SHARED_LIBS=ON '
+# required to install extra tools in bin/
+configopts += '-DLLVM_INSTALL_UTILS=ON -DLLVM_TARGETS_TO_BUILD=X86 -DLLVM_ENABLE_ZLIB=ON -DLLVM_ENABLE_RTTI=ON '
+configopts += '-DCMAKE_BUILD_TYPE=Release '
```
Comment (lexming) [inline easybuild/easyconfigs/l/LLVM/LLVM-10.0.0-GCCcore-8.3.0.eb:36]: Recently we added a specific variable to automatically handle `BUILD_TYPE`
```suggestion
build_type = 'Release'
```

**Comment by Falkor** [comment]
Test report by @ULHPC
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
iris-040 - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 3.7.4
See https://gist.github.com/7d8fb7cf39d08269ab1d65eff618297f for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node314.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/d09c7f23ec6b821f789b9b9ae6cd1ef4 for a full test report.

**Comment by lexming** [comment]
Going in, thanks @Falkor !

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3133.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/9986286831304b6ae17b4ee4e21e8b8c for a full test report.

**Pull request merged**